Detectors/ITSMFT/MFT/base/include/MFTBase/Barrel.h

```diff
@@ -25,7 +25,7 @@ class Barrel : public TNamed
 
  public:
   Barrel() = default;
-  ~Barrel() = default;
+  ~Barrel() = override;
```

Comment:
override is only needed for virtual classes. Do you need Barrel to be virtual (aka do you need to inherit from TNamed?).